### DISCUSSION TITLE
Testing w/ Cypress 10

### DISCUSSION THREAD

**Discussion by bokimgem** [react-dropzone/react-dropzone#1240] [General]
Hello, I'm trying to do some e2e testing using react-dropzone but having no luck getting the onDrop callback to fire. Any suggestions?

**Comment by bokimgem** [comment]
Was able to get it to work in Cypress Electron by calling `selectFile` command multiple times. Calling it once, for some reason or another, does not do the trick.

**Comment by rolandjitsu** [comment]
@bokimgem unfortunately I'm not very familiar with that testing tool. If you could provide a simple example I could try to have a look.
